proto/manifest.proto

```diff
@@ -12,22 +12,26 @@ message Resource {
     // Path specifies the path from the bundle root. If more than one
     // path is present, the entry may represent a hardlink, rather than using
     // a link target. The path format is operating system specific.
+    // Path MAY be an absolute path, but it MUST be interpreted with the bundle root.
     repeated string path = 1;
 
     // NOTE(stevvooe): Need to define clear precedence for user/group/uid/gid precedence.
 
     // Uid specifies the user id for the resource. A string type is used for
     // compatibility across different OS.
+    // On Unix, uid MUST be a positive decimal integer or an empty string.
     string uid = 2;
```

Comment:
Let's just change these to integers. Windows aren't gonna use these, so they'll always be integers. Use `int64`.